### PR TITLE
Rename `times` to `repeat`.

### PR DESCRIPTION
**Pull request by andersio**
Related issue: #118

### CODE REVIEW

File: Sources/Deprecations+Removals.swift [hunk 1]
```diff
@@ -279,6 +288,18 @@ extension SignalProducerProtocol {
 
 	@available(*, unavailable, message:"This SignalProducer may emit errors which must be handled explicitly, or observed using `startWithResult(_:)`.")
 	public func startWithNext(_ next: (Value) -> Void) -> Disposable { fatalError() }
+
+	@available(*, unavailable, renamed:"repeat(_:)")
```
Comment (NachoSoto) [inline Sources/Deprecations+Removals.swift]: I agree `repeat` is better.

File: Sources/Deprecations+Removals.swift [hunk 2]
```diff
@@ -207,6 +207,15 @@ extension SignalProtocol {
 
 	@available(*, unavailable, message: "This Signal may emit errors which must be handled explicitly, or observed using `observeResult(_:)`")
 	public func observeNext(_ next: (Value) -> Void) -> Disposable? { fatalError() }
+
+	@available(*, unavailable, renamed:"catch(_:)")
+	public func flatMapError<F>(_ handler: @escaping (Error) -> SignalProducer<Value, F>) -> Signal<Value, F> { fatalError() }
```
Comment (NachoSoto) [inline Sources/Deprecations+Removals.swift]: `flatMapError` is far more descriptive. `catch` is a well known word but it doesn't carry the extra information that this does, which draws a parallel with `flatMap`.

File: Sources/Deprecations+Removals.swift [hunk 3]
```diff
@@ -207,6 +207,15 @@ extension SignalProtocol {
...
+
+	@available(*, unavailable, renamed:"try(_:)")
+	public func attempt(_ operation: @escaping (Value) -> Result<(), Error>) -> Signal<Value, Error> { fatalError() }
+
+	@available(*, unavailable, renamed:"tryMap(_:)")
+	public func attemptMap<U>(_ operation: @escaping (Value) -> Result<U, Error>) -> Signal<U, Error> { fatalError() }
```
Comment (NachoSoto) [inline Sources/Deprecations+Removals.swift]: These names are meant to match the `Result` counterparts.

File: Sources/Deprecations+Removals.swift [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import Foundation
-import enum Result.NoError
+import Result
```
Comment (ikesyo) [inline Sources/Deprecations+Removals.swift]: Should this be reverted too?

**Pull request merged**